app/src/apollo/index.js

```diff
@@ -0,0 +1,18 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client';
+import { HTTP_URI } from '../config';
+
+const client = new ApolloClient({
+  uri: HTTP_URI,
+  cache: new InMemoryCache(),
+  defaultOptions: {
+    query: {
+      fetchPolicy: 'network-only',
+      errorPolicy: 'all',
+    },
+    mutate: {
+      errorPolicy: 'all',
+    },
+  },
+});
+
+export default client;
```

Comment:
So we are still going to use GraphQL and Apollo?

Reply:
Yes, if no one has any issue with it, I elaborated on my design doc pros and cons of the tech stack.

I did however refactored and integrated the backend service into App Engine, but those changes are available on a separate PR.